tests/postgres_tests/test_ranges.py

```diff
@@ -193,6 +196,45 @@ def test_date_range_contains(self):
                     [self.obj, self.aware_obj],
                 )
 
+    def test_auto_field_contained_by(self):
+        objs = CharFieldModel.objects.bulk_create([
+            CharFieldModel(field=str(i)) for i in range(1, 5)
+        ])
+        self.assertSequenceEqual(
+            CharFieldModel.objects.filter(id__contained_by=NumericRange(1, 3)),
+            [objs[0], objs[1]]  # [id=1, id=2]
+        )
+
+    def test_big_auto_field_contained_by(self):
+        objs = BigAutoFieldModel.objects.bulk_create([
+            BigAutoFieldModel() for i in range(1, 5)
+        ])
+        self.assertSequenceEqual(
+            BigAutoFieldModel.objects.filter(id__contained_by=NumericRange(2, 5)),
+            [objs[1], objs[2], objs[3]]  # [id=2, id=3, id=4]
+        )
+
+    def test_fields_contained_by(self):
+        objs = ContainedByTestModel.objects.bulk_create([
+            ContainedByTestModel(
+                small_integer_field=i,
+                decimal_field=i,
+            )
+            for i in range(1, 10)
+        ])
+        self.assertSequenceEqual(
+            ContainedByTestModel.objects.filter(id__contained_by=NumericRange(5, 8)),
+            [objs[4], objs[5], objs[6]]  # [id=5, id=6, id=7]
+        )
+        self.assertSequenceEqual(
+            ContainedByTestModel.objects.filter(small_integer_field__contained_by=NumericRange(4, 6)),
+            [objs[3], objs[4]]  # [id=4, id=5]
+        )
+        self.assertSequenceEqual(
+            ContainedByTestModel.objects.filter(decimal_field__contained_by=NumericRange(6, 9)),
+            [objs[5], objs[6], objs[7]]  # [id=6, id=7, id=8]
+        )
```

Comment:
These tests should probably be broken in separate methods?

Also the `# [id=5, id=6, id=7]` comments should be removed since they will only be true if the test is run in isolation since PostgreSQL doesn't reset sequences on rollback.